src/lib/layout/containerHeader.svelte

```diff
@@ -86,12 +86,14 @@
 
 <!-- Show only if on Cloud, alerts are enabled, and it isn't a project limited service -->
 {#if isCloud && showAlert}
-    {#if $readOnly}
-        {@const services = overflowingServices
-            .map((s) => {
-                return s.name.toLocaleLowerCase();
-            })
-            .join(', ')}
+    <!-- some services are above limit -->
+    {@const services = overflowingServices
+        .map((s) => {
+            return s.name.toLocaleLowerCase();
+        })
+        .join(', ')}
+
+    {#if $readOnly && services.length}
```

Comment:
This might still be a problem because `$readOnly` is only true when there are failed invoices, right?

Also, does this mean some of these things don't show if they don't have access (non owner role in the org)?!

Reply:
> Yea, seems we should remove readOnly check as we do overflowing services check.

Ya makes sense.